### PR TITLE
Improve toml fuzzer

### PR DESCRIPTION
**Pull request by alanhdu**
This improves the TOML fuzzer by:
- checking that serializing and deserializing a valid TOML value does not change anything
- Uses https://github.com/uiri/toml-test as a seed corpus, giving us higher coverage from the start and drastically improving fuzzing speed with its smaller test cases (on my laptop, I get 3.5x executions / second).

### CODE REVIEW

File: toml/read_toml.rs [hunk 1]
```diff
@@ -5,6 +5,8 @@ extern crate toml;
 
 fuzz_target!(|data| {
     if let Ok(data) = toml::from_slice::<toml::Value>(data) {
-        let _ = toml::to_string(&data);
+        let s = toml::to_string(&data).unwrap();
+        let copy = toml::from_str(&s).unwrap();
+        assert_eq!(data, copy);
```
Comment (frewsxcv) [inline toml/read_toml.rs]: maybe this file should be renamed to `read_write_toml.rs`? what do you think?

Comment (killercup) [inline toml/read_toml.rs]: I'd vote for `toml_roundtrip.rs`

**Comment by alanhdu** [comment]
I used `roundtrip_toml`, to be consistent with the verb_noun naming scheme that seems to be in place.

File: toml/read_toml.rs (hunk 1 above)
Comment (alanhdu) [inline toml/read_toml.rs]: I went with roundtrip_toml, to be consistent with the verb_noun naming scheme that we seem to be using.

**Comment by frewsxcv** [comment]
thanks!

bors r+

if you find any bugs via fuzzing, feel free to add to https://github.com/rust-fuzz/trophy-case

**Pull request merged**